packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -305,6 +305,7 @@ const ExpandableItemBase = kind({
 				} else if (isDown(keyCode) && wouldDirectionLeaveContainer('down', target)) {
 					if (lockBottom) {
 						ev.nativeEvent.stopImmediatePropagation();
+						ev.preventDefault();
```

Comment:
do we need this for 'up' key too?

Reply:
Good point! I just updated to fix `up` key too. The issue happens `ExpandableItem` is located on the top of a viewport of a scroller and the currently focused item is the first one.